tests/downstream_more_column/run.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+set -eu
+
+cur=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+source $cur/../_utils/test_prepare
+WORK_DIR=$TEST_DIR/$TEST_NAME
+db="downstream_more_column"
+tb="t1"
+
+function get_master_status() {
```

Comment:
maybe we could use get_master_status from upstream_switch/lib.sh

Reply:
Add in test_prepare so we can reuse it.